contrib/time_series_expectations/time_series_expectations/expectations/expect_batch_row_count_to_match_prophet_date_model.py

```diff
@@ -0,0 +1,181 @@
+from typing import Dict, Optional
+
+import pandas as pd
+
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.data_context.util import file_relative_path
+from great_expectations.execution_engine import (
+    ExecutionEngine,
+)
+from great_expectations.expectations.expectation import TableExpectation
+from time_series_expectations.expectations.prophet_model_deserializer import (
+    ProphetModelDeserializer,
+)
+
+with open(file_relative_path(__file__, "example_prophet_date_model.json")) as f_:
+    example_prophet_date_model_json = f_.read()
+
+
+class ExpectBatchRowCountToMatchProphetDateModel(TableExpectation):
+    """This Expectation checks to see if the number of rows in a Batch matches the predictions of a prophet model for a given date.
+
+    expect_batch_row_count_to_match_prophet_date_model is a [TableExpectation](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_table_expectations)
+
+    Args:
+        date (str):
```

Comment:
❤️ Thank you for these docstrings!

Reply:
Thanks for giving me an example to work from!